real_estate/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+
+from . import controllers
+from . import models
```

Comment:
Final new line is missing
https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/missing-final-newline.html